spec/features/advanced_search_spec.rb

```diff
@@ -38,20 +38,58 @@
       @community.delete
     end
     it "finds uuid" do
-      search("uuid:394266f0-0e4a-42e6-a199-158165226426")
+      search('all_fields', "uuid:394266f0-0e4a-42e6-a199-158165226426")
       expect(page).to have_content('Bison sculpture at the entrance to the USGS Ice Core Lab')
     end
   end
 
+  describe "Check thesis date search" do
+    before do
+      Collection.delete_all
+      @community = Collection.new(title:'FGSR').tap do |c|
+        c.apply_depositor_metadata('dittest@ualberta.ca')
+        c.is_community = true
+        c.is_official = true
+        c.fedora3uuid = 'uuid:39331f1f-769d-4c2a-a103-416c285d01fc'
+        c.save
+      end
+      @collection = Collection.new(title:'Theses').tap do |c|
+        c.apply_depositor_metadata('dittest@ualberta.ca')
+        c.is_official = true
+        c.fedora3uuid = 'uuid:7af76c0f-61d6-4ebc-a2aa-79c125480269'
+        c.save
+      end
+      GenericFile.delete_all
+      Rake::Task.define_task(:environment)
+      Rake::Task["migration:eraitem"].invoke('spec/fixtures/migration/test-metadata/thesis-metadata')
```

Comment:
I'm not keen on using the migration to set up tests.  It relies on Fedora 3 being accessed which won't be possible in all test environments + it's time consuming.